zendesk/user.go

```diff
@@ -135,4 +135,25 @@ func (z *Client) CreateUser(ctx context.Context, user User) (User, error) {
 	return result.User, nil
 }
 
+// CreateOrUpdateUser creates new user
```

Comment:
This method should probably be added to the [UserAPI interface](https://github.com/nukosuke/go-zendesk/blob/master/zendesk/user.go#L81-L85). You'll need to regenerate the mocked client after doing that. To regenerate install [mockgen](https://github.com/golang/mock#installation) and run `go generate ./...`

Reply:
done